code/modules/mob/hear_say.dm

```diff
@@ -167,7 +167,22 @@
 
 /mob/proc/hear_sleep(var/message)
 	var/heard = ""
+	var/opentag = 0
+	var/closetag = 0
 	if(prob(15))
+		while(1) //Remove the HTML formatting so people don't hear colour
```

Comment:
@MarcellusPye https://pastebin.com/raw/gHGi3Rhk if you need to test with script that has a decent amount of formatting.